recipes/Media/ASR/seq2seq/hparams/train.yaml

```diff
@@ -0,0 +1,149 @@
+# ################################
+# Model: CRDNN + DNN + CTC + Softmax
+# Authors modifier: Gaelle Laperriere
+# ################################
+
+# ------ Paths and parameters
+
+# Seed needs to be set at top of yaml, before objects with parameters are made.
+seed: 4242
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/media_ASR/<seed>
+cer_file_test: !ref <output_folder>/cer_test.txt
+ctc_file_test: !ref <output_folder>/ctc_test.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files:
+data_folder: !PLACEHOLDER
+csv_train: !ref <data_folder>/train.csv
+csv_valid: !ref <data_folder>/dev.csv
+csv_test: !ref <data_folder>/test.csv
+
+# Data parameters:
+# With data_parallel batch_size is split into N jobs.
+# With DDP batch_size is multiplied by N jobs.
+batch_size: 4
+test_batch_size: 2
+# We remove utterances longer than 90s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 90.0
+avoid_if_smaller_than: 0.0
+dataloader_options:
+    batch_size: !ref <batch_size>
+    num_workers: 3
+    shuffle: True
+test_dataloader_options:
+    batch_size: !ref <test_batch_size>
+    num_workers: 3
+
+# Feature parameters:
+sample_rate: 16000
+n_mels: 80
+n_fft: 400
+f_min: 0
+
+# Training parameters:
+number_of_epochs: 26 #100
+lr: 1
+annealing_factor: 0.8
+improvement_threshold: 0.0025
+patient: 0
+sorting: random
```

Comment:
Hum, are you sure about the random? Usually using ascending is much faster and provides good results as well.

Reply:
Not sure, I kept it from another yaml where it was necessary. If I write "ascending" here, the results in the README won't be relevant. Must I change "random" to "ascending"?

Reply:
I would encourage you in re-running the exp with ascending honestly. It usually is much master.

Reply:
Ok I will. I'll update the results once it is done.